src/styles/getModalStyles/getModalStyles.js

```diff
@@ -16,7 +16,7 @@ export default (type, windowDimensions, popoverAnchorPosition = {}, containerSty
     let animationOut;
     let hideBackdrop = false;
     let shouldAddBottomSafeAreaPadding = false;
-    let shouldAddTopSafeAreaPadding = false;
+    const shouldAddTopSafeAreaPadding = shouldModalAddTopSafeAreaPadding[type] || false;
```

Comment:
This where `shouldAddTopSafeAreaPadding` will have according to what value we want from specific platform.